cmake/developer_package/frontends/frontends.cmake

```diff
@@ -155,11 +155,27 @@ macro(ov_add_frontend)
         list(APPEND PROTO_HDRS "${OUTPUT_PB_HEADER}")
     endforeach()
 
+    file(GLOB flatbuffers_schema_files ${frontend_root_dir}/src/schema/*.fbs)
```

Comment:
Do we really need to have this code in common place? Or is it a TF lite specific?

Reply:
This is generic code, which takes .proto or .fbs files from frontends and automatically compiles them using proper compiler (protoc / flatc)